### ISSUE TITLE
WCAG: Compatible

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/hourglass#331]
# Summary

This review considered:
- Name, Role, Value
- Status Messages

# Audit Results

## Name, Role, Value
The email input at the bottom of the homepage has both an aria label and a placeholder, both of which are read by the screen-reader as already discussed in #329

The same is happening with the "quick search" input and with filters so I think all inputs need to be reviewed for this issue.

## Status Messages
this has already been discussed in regards to log in failure messages see #333
